extensions/binding/org.eclipse.smarthome.binding.digitalstrom/ESH-INF/thing/dssBridge.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please apply formatter (best on all java and xml files, so I won't mention it any further)